src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -374,8 +401,18 @@ def get_suggestion(self, random_search=False):
         total_params : dict
             parameter suggestion
         """
-
-        rval = self.rval
+        if self.parallel == True and len(self.running_data):
+            self.CL_rval = copy.deepcopy(self.rval)
```

Comment:
Every time we copy a "self.rval" to "self.CL_rval". Shall we need to consider the memory recycle here?